modules/performance/extension/siddhi-execution-performance/component/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.wso2.extension.siddhi.execution.performance</groupId>
         <artifactId>siddhi-execution-performance-parent</artifactId>
-        <version>4.4.0-m2-SNAPSHOT</version>
+        <version>4.4.0-m3-SNAPSHOT</version>
```

Comment:
You don't need to change the version

Reply:
This has to be changed since there is a mismatch between the parent pom file version which is 4.4.0-m3-SNAPSHOT.